### DISCUSSION TITLE
Custom ticker

### DISCUSSION THREAD

**Discussion by goldenratio** [jnsmalm/pixi3d#127] [Q&A] [answered]
Hey, 
I have custom ticker (update loop) that powers pixi, spine and pixi3d gltf animations (in play method, I can pass my own ticker). Noticed that PostProcessing and PickingInteraction uses pixi's shared ticker. Will it cause any side effects? is it possible use my own custom ticker.

**Comment by jnsmalm** [comment, marked as answer]
It should not cause any side effects. If it does, please let me know and I'll fix it.
